Python/Module1_GettingStartedWithPython/Informal_Intro_Python.md

```diff
@@ -19,6 +19,39 @@ jupyter:
 <!-- #endraw -->
 
 # An Informal Introduction to Python
+
+<div class="alert alert-warning">
+
+**Before You Start This Section**: 
+
+In the following section we will be using IPython or a Jupyter notebook to run our code.
+Presently, there is an incompatibility with these programs and a Python package called `jedi`, which typically is responsible for performing auto-completions in our code (when prompted by hitting `<TAB>`, which we will be doing below).
+It is really useful!
+
+First, let's check to see if we have an incompatible version of `jedi` installed.
+In your terminal (before starting a Python/IPython/Jupyter session), run
+    
+```
+conda list
+```
+
+And look for the line that starts with `jedi`
```

Comment:
does the windows version have grep built in? would be a lot easier to

```shell
conda list | grep jedi
```

if so

Reply:
Mine does, but I have no clue if it is because I installed git-bash or something. I know some students use powershell and anaconda terminal, so I figure... just let 'em scroll 😬